### PR TITLE
Add space member filter

### PR DESCRIPTION
**Pull request by JanAckermann**
## Related Issue

- Fixes https://github.com/owncloud/web/issues/7746

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

**Review by kulmann — CHANGES_REQUESTED** [review]
The layout feels very jumpy, could you look into that? Do we really need the label above the input field (that's what creates the jumpiness). Ideally the `x` close icon for the filter should be in the same place as the 🔎 icon, so that you can toggle back and forth without moving the mouse.

**Review by kulmann — CHANGES_REQUESTED** [review]
Also, could you highlight the search term in the filtered member names? Same as e.g. in the user management app.

**Comment by JanAckermann** [comment]
@kulmann thx for the input, done

File: packages/web-app-files/src/components/SideBar/Shares/SpaceMembers.vue [hunk 1]
```diff
@@ -1,8 +1,37 @@
 <template>
   <div id="oc-files-sharing-sidebar" class="oc-position-relative">
-    <div class="oc-flex">
-      <h3 v-translate class="oc-text-bold oc-text-medium oc-m-rm">Members</h3>
-      <oc-contextual-helper v-if="helpersEnabled" class="oc-pl-xs" v-bind="spaceAddMemberHelp" />
+    <div class="oc-flex oc-flex-between oc-flex-center oc-flex-middle">
+      <div class="oc-flex">
```
Comment (kulmann) [inline packages/web-app-files/src/components/SideBar/Shares/SpaceMembers.vue:4]: If you give this div a padding top and bottom of 6px (or more) each, there is no jumpiness at all anymore.

Comment (kulmann) [inline packages/web-app-files/src/components/SideBar/Shares/SpaceMembers.vue:4]:
```suggestion
      <div class="oc-flex oc-py-s">
```

**Review by kulmann — APPROVED** [review]
nice! 😍

**Pull request merged**